UefiCpuPkg/PiSmmCpuDxeSmm/SmmProfile.c

```diff
@@ -884,33 +884,6 @@ CheckFeatureSupported (
     }
   }
 
-  if (mXdSupported) {
-    AsmCpuid (CPUID_EXTENDED_FUNCTION, &RegEax, NULL, NULL, NULL);
-    if (RegEax <= CPUID_EXTENDED_FUNCTION) {
-      //
-      // Extended CPUID functions are not supported on this processor.
-      //
-      mXdSupported = FALSE;
-      PatchInstructionX86 (gPatchXdSupported, mXdSupported, 1);
-    }

```

Comment:
currently, cet feature is still in the function, which is also the software-domain feature and we have confirmed it's consistent both inside and outside SMM.  it can be also moved out from the function, but it's not related to the standalone mm support. 

once the CET moved out, i agree we can rename to the DetectFeaturesPerAp. but for now, i prefer keep it as is

Reply:
synced with Ray, agree above